libraries/standard/http/lexicon.txt

```diff
@@ -55,9 +55,13 @@ iso
 lastheaderfieldlen
 lastheadervaluelen
 latin
+len
```

Comment:
What is this file, I don't remember seeing this before.

Reply:
This is just for our spell checks in CI where we specify each word in the codebase.